[2025-08-27T14:54:45.023+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-08-27T14:54:45.076+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: s3_upload_pipeline_v1.create_and_upload_file_to_s3 manual__2025-08-27T14:54:36.896552+00:00 [queued]>
[2025-08-27T14:54:45.091+0000] {taskinstance.py:2076} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: s3_upload_pipeline_v1.create_and_upload_file_to_s3 manual__2025-08-27T14:54:36.896552+00:00 [queued]>
[2025-08-27T14:54:45.092+0000] {taskinstance.py:2306} INFO - Starting attempt 1 of 1
[2025-08-27T14:54:45.130+0000] {taskinstance.py:2330} INFO - Executing <Task(_PythonDecoratedOperator): create_and_upload_file_to_s3> on 2025-08-27 14:54:36.896552+00:00
[2025-08-27T14:54:45.147+0000] {warnings.py:112} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=170) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2025-08-27T14:54:45.150+0000] {standard_task_runner.py:63} INFO - Started process 172 to run task
[2025-08-27T14:54:45.151+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 's3_upload_pipeline_v1', 'create_and_upload_file_to_s3', 'manual__2025-08-27T14:54:36.896552+00:00', '--job-id', '29', '--raw', '--subdir', 'DAGS_FOLDER/s3_upload_dag.py', '--cfg-path', '/tmp/tmp5k58ixk1']
[2025-08-27T14:54:45.156+0000] {standard_task_runner.py:91} INFO - Job 29: Subtask create_and_upload_file_to_s3
[2025-08-27T14:54:45.288+0000] {task_command.py:426} INFO - Running <TaskInstance: s3_upload_pipeline_v1.create_and_upload_file_to_s3 manual__2025-08-27T14:54:36.896552+00:00 [running]> on host 9c188e6f7874
[2025-08-27T14:54:45.474+0000] {taskinstance.py:2648} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='s3_upload_pipeline_v1' AIRFLOW_CTX_TASK_ID='create_and_upload_file_to_s3' AIRFLOW_CTX_EXECUTION_DATE='2025-08-27T14:54:36.896552+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-08-27T14:54:36.896552+00:00'
[2025-08-27T14:54:45.479+0000] {taskinstance.py:430} INFO - ::endgroup::
[2025-08-27T14:54:45.482+0000] {logging_mixin.py:188} INFO - Fajl 'test_iz_***a.txt' kreiran u kontejneru.
[2025-08-27T14:54:45.521+0000] {credentials.py:1147} INFO - Found credentials in environment variables.
[2025-08-27T14:54:46.196+0000] {logging_mixin.py:188} INFO - Uploadujem fajl u bucket 'milos-de-portfolio-bucket-2025'...
[2025-08-27T14:54:47.121+0000] {logging_mixin.py:188} INFO - Upload uspe≈°an!
[2025-08-27T14:54:47.125+0000] {python.py:237} INFO - Done. Returned value was: None
[2025-08-27T14:54:47.127+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-08-27T14:54:47.150+0000] {taskinstance.py:1206} INFO - Marking task as SUCCESS. dag_id=s3_upload_pipeline_v1, task_id=create_and_upload_file_to_s3, run_id=manual__2025-08-27T14:54:36.896552+00:00, execution_date=20250827T145436, start_date=20250827T145445, end_date=20250827T145447
[2025-08-27T14:54:47.214+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-08-27T14:54:47.249+0000] {taskinstance.py:3503} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2025-08-27T14:54:47.254+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
